include/wx/generic/grid.h

```diff
@@ -16,13 +16,19 @@
 #if wxUSE_GRID
 
 #include "wx/hashmap.h"
-
+#include "wx/overlay.h"
 #include "wx/scrolwin.h"
 
 #if wxUSE_STD_CONTAINERS_COMPATIBLY
     #include <iterator>
 #endif
 
+#if defined(__WXGTK3__)
```

Comment:
I'd like to test this under Mac to see if it also works with native overlay implementation there. It really ought to, and then we wouldn't need to have this GTK-specific test and could just use `wxHAS_NATIVE_OVERLAY`.

Although, OTOH, I al;so wonder why shouldn't we always use overlay, i.e. why shouldn't it work even with the generic implementation?

Reply:
I'm focusing on getting wxOverlay working decently under GTK3, i'll test with the generic version later.